src/main/java/com/teamc/fms/service/impl/TenantServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package com.teamc.fms.service.impl;
+
+import com.teamc.fms.entity.Tenant;
+import com.teamc.fms.repo.TenantRepository;
+import com.teamc.fms.service.TenantService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+public class TenantServiceImpl implements TenantService {
+
+    private TenantRepository tenantRepository;
+
+
+    @Override
+    public ResponseEntity<Tenant> save(Tenant tenant) {
+
+        Tenant tenant1 =tenantRepository.save(tenant);
```

Comment:
done use tenant1, use descriptive name
Do not use ResponseEntity at service layer